raster/src/main/scala/geotrellis/raster/mapalgebra/focal/Kernel.scala

```diff
@@ -68,14 +68,15 @@ object Kernel {
 
     val rad2 = rad*rad
 
-    var r = 0
-    var c = 0
-    while(r < size) {
-      while(c < size) {
-        output.set(c, r, if (r * r + c * c < rad2) 1 else 0)
+    val w = size/2
```

Comment:
Good catch on this. But this looks like some pretty neglected interface. 'cellWidth' parameter is not used. Its not clear why you have to give `size` and `rad`. I would suggest to use `geotrellis.raster.mapalgebra.focal.Circle` `Neighborhood` to stamp the tile and reduce the parameter to a single radius, which is in pixels.